### DISCUSSION TITLE
Notification timeout

### DISCUSSION THREAD

**Discussion by iaroki** [folke/noice.nvim#113] [Q&A] [answered]
Hello,
Is there any option to set timeout for notify?
I tried setting it for vim-notify itself but it didn't work. Noticed timeout option for mini backend so had to rollback to it but default timeout of 2sec for mini just enough for me.
It would be great to somehow set timeout for notify.

**Comment by folke** [comment, marked as answer]
You need to configure that in notify setup.

```lua
  require("notify").setup({
    timeout = 10000,
})
```

**Reply by iaroki** [reply]
Yeah, I was trying this and didn't notice much difference. 
Notifications is faster by setting this value pretty low but still do not exceed some level. Like setting this to 1000 or 500 still getting notifications for like 2 seconds.

But anyway thanks for answer I will do dig into vim-notify more.

**Reply by folke** [reply]
I think that's because of the animations. The timeout probably only starts after all animations finsihed.
You could set `stages='static'` in notify to skip animations. That will probably honour the timeout

**Comment by StitiFatah** [comment]
What about if one is using `mini` instead of `notify` ?
